android/app/src/main/java/network/mysterium/MainActivity.kt

```diff
@@ -132,7 +132,7 @@ class MainActivity : AppCompatActivity(), ActivityCompat.OnRequestPermissionsRes
     }
 
     override fun onRequestPermissionsResult(requestCode: Int, permissions: Array<out String>, grantResults: IntArray) {
-        notifications.onRequestPermissionsResult(requestCode, permissions, grantResults)
+        //notifications.onRequestPermissionsResult(requestCode, permissions, grantResults)
     }
```

Comment:
Please leave a comment why this part of the code is commented, if it's not used - delete it

Reply:
Currently this class is unused because it's from previous app version. I commented the line for successful build. In future all classes from previous version except node core will be deleted from app.